src/bin/flamegraph.rs

```diff
@@ -20,10 +20,15 @@ struct Opt {
     #[structopt(long = "cp")]
     cp: bool,
 
-    /// Colors are keyed by function name hash
+    /// Colors are selected by hashing the function name, weighting the first characters more
+    /// heavily
     #[structopt(long = "hash")]
     hash: bool,
 
+    /// Colors are selected chaotically but the color of a function does not change between runs
+    #[structopt(long = "deterministic")]
```

Comment:
This probably `conflicts_with = "hash"`.

Reply:
Even though it's probably unnecessary, I'd also add `conflicts_with = "hash"` to this option as well.